src/Task1/SelectGenre.jsx

```diff
@@ -29,44 +34,48 @@ const SelectGenre = () => {
           }}
         >
           Select Genre
-          <span style={{ clipPath: clip }} className="arrow">
-            ok
-          </span>
+          <span className={`arrow ${isopen ? "opened" : "notopen"}`}>ok</span>
```

Comment:
`nit:` This course is not about CSS, code quality, general coding principles and stuff like that, but I would suggest a bit more grammatically correct, simpler, and more consistent names for your "openness" class-names - `.open` and `.closed`

Reply:
Hmmm, yes I fix